src/main/java/com/googlecode/download/maven/plugin/internal/HttpFileRequester.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2009-2016 Marc-Andre Houle and Red Hat Inc
```

Comment:
If you created this class, please make sure the Copyright shows the right date and the right owner (you or your employer depending on your agreement)

Reply:
@mickaelistria OK. Corrected.

Reply:
Are you working for the Apache Foundation? If not, you're not supposed to cease your copyright.

You should either put your name or the name of your company or employer, depending on the context you're doing this. It could be something like `Copyright (c) 2018, OK2Consulting`.

Reply:
@ok2c I am contributing this code as an ASF member, not as an employee of OK2 Consulting.

Reply:
Ok, I do not know the terms of Apache Foundation regarding copyrights (are members allow to copyright their work on the name of the Foundation?). I know that for Eclipse Foundation, members/committers have to use their own or employer copyright and are not permitted to set Copyright to Eclipse Foundation.
If you confirm me that it's a practice that's accepted by the Apache Foundation, then I'm fine with it and will resolve.

Reply:
@ok2c I am not a legal expert, but as per CLA I have with ASF I can grant copyright of my code contributions and IP therewith to ASF.

Reply:
Ok, thanks!